styles.css

```diff
@@ -0,0 +1,43 @@
+
+@import url('https://fonts.googleapis.com/css2?family=Roboto&display=swap');
+
+body {
+  font-size: 20px;
+  background-color: rgba(26, 25, 32, 0.991);
+  color: beige;
+}
+
+.navbar {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  /* background-color: #333; */
+
+  /* color: white; */
+}
+
+.logo {
+  text-align: left;
+  font-size: 50px;
+  text-decoration: none;
+  color: greenyellow;
+}
+
+.bx {
+  font-size: 50px;
+  margin-right: 6vw;
+}
+
+.description {
+  text-align: center;
+  padding: 30vh;
+  font-size: 20px;
+}
```

Comment:
- [ ] Good job so far but I noticed that you did follow any template design on this project , but your text are settled well , I encourage you to use this [Choose one template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122) to match all requirements for this milestone of your project :wink: 
This is how your design looks like point_down: 
![aimer](https://user-images.githubusercontent.com/88894525/199202777-2937efd3-e7a8-4e2c-b227-d1b1ed1543f0.png)